### PR TITLE
[Delivers #98910592] warning banner for sandbox sites.

### PR DESCRIPTION
**Pull request by annalee**
"As a SANDBOX USER, I can see a reminder that the website is a sandbox so that I don't accidentally make live requests."
screen shot 2015-07-17 at 5 34 49 pm
screen shot 2015-07-17 at 5 35 11 pm

### CODE REVIEW

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -4,6 +4,11 @@
   html5up.net | @n33co
   Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
 %html{lang: 'en'}
+  - unless ENV["DISABLE_SANDBOX_WARNING"]
+    %div{class: "sandbox-banner"}
+      %span
+        This sandbox site is for testing and training purposes only. To view and make live requests,
+        %a{href: "https://cap.18f.gov"} go to the live site.
```
Comment (afeld) [inline app/views/home/index.html.haml]: Mind making a shared partial for this, and using it on this page and in the partial below?

http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

File: app/assets/stylesheets/common/header.scss [hunk 2]
```diff
@@ -22,6 +22,16 @@ header {
...
 
+.sandbox-banner {
+  width: 100%;
+  display: block;
+  background: #fff2bb;
+  border: 1px solid #d9d9d9;
+  border-top: none;
+  text-align: center;
+  padding: 10px 10px;
+}
```
Comment (afeld) [inline app/assets/stylesheets/common/header.scss]: Mind making a standalone stylesheet for the banner stuff?

File: spec/features/sandbox_warning_spec.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    end
+  end
+
+  context 'hides sandbox warning in production' do
+    before do
+      ENV["DISABLE_SANDBOX_WARNING"] = "true"
+    end
+    after do
+      ENV["DISABLE_SANDBOX_WARNING"] = nil
+    end
```
Comment (afeld) [inline spec/features/sandbox_warning_spec.rb]: Minor: use `with_feature`.

**Comment by phirefly** [comment]
:shipit:

**Pull request merged**